contracts/Proxy/BorrowerOperationsScript.sol

```diff
@@ -4,42 +4,88 @@ pragma solidity 0.6.11;
 
 import "../Dependencies/CheckContract.sol";
 import "../Interfaces/IBorrowerOperations.sol";
+import "../Dependencies/IERC20.sol";
 
 
-contract BorrowerOperationsScript is CheckContract {
```

Comment:
Why are you removing this? Shouldn’t you remove the import above too?

Reply:
Actually I forgot to remove the import. None of the inherited functions are used so any specific reason for CheckContract?

Reply:
Well, before it was used:
```
checkContract(address(_borrowerOperations));
```
We do that as a sanity check in all of our contracts, and it was also recommended by auditors, why should we remove it?

Reply:
I guess that was a mistake. will fix that.